cfgov/core/management/commands/generate_streamfield_docs.py

```diff
@@ -0,0 +1,127 @@
+from collections import defaultdict
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from wagtail.wagtailcore.blocks import StreamBlock
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailcore.models import Page
+
+
+class Command(BaseCommand):
+    help = 'Generate Wagtail StreamField block documentation'
+
+    def handle(self, *args, **options):
+        blocks, models = get_blocks_and_models()
+
+        template = loader.get_template('core/docs/streamfield-blocks.md')
+        md = template.render({
+            'blocks': blocks,
+            'models': models,
+        })
+
+        self.stdout.write(md)
```

Comment:
I think it would be nice if this had an option to write out the output with `-o` or something like that instead of us having to rely entirely on shell output redirection. I could see that potentially getting tricky if this is scripted in, say, yaml.

Reply:
Nice suggestion, implemented in 71d0ed9.